en/advanced_config/pid_tuning_guide_fixedwing.md

```diff
@@ -131,7 +131,7 @@ fixed wing setups and usually does not require adjustment.
 All L1 parameters are described [here](../advanced_config/parameter_reference.md#fw-l1-control).
 
 - [FW_L1_PERIOD](../advanced_config/parameter_reference.md#FW_L1_PERIOD) - This is the L1 distance and defines the tracking
-  point ahead of the aircraft its following. A value of 25 meters
+  point ahead of the aircraft it is following . A value of 25 meters

```

Comment:
Additional space before full stop is not needed (otherwise good fix, though I would have simply changed to "it's")